Sources/LocalLLMClientCore/LLMSession.swift

```diff
@@ -302,6 +302,28 @@ public extension LLMSession {
         public func downloadModel(onProgress: @Sendable @escaping (Double) async -> Void = { _ in }) async throws {
             try await downloader.download(onProgress: onProgress)
         }
+
+        public static func pruneModels(in url: URL = FileDownloader.defaultRootDestination, excluding: [Model] = []) throws {
```

Comment:
Accepting any `Model` is merely for convenience. We're later ensuring that skipped models are `DownloadModel`s (which are the only models to store files on the FS).